testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -121,9 +124,6 @@ def test_triangular_matrix(self):
                        reason="Not yet supported on Windows.")
```

Comment:
Does it still fail on Windows, even after the fix?

Reply:
ya, it fails on Windows since `strict=True`. Switching it to false should do the trick.